src/main/java/org/apache/accumulo/testing/randomwalk/shard/Grep.java

```diff
@@ -88,13 +88,11 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
     bs.close();
 
     if (!documentsFoundInIndex.equals(documentsFoundByGrep)) {
-      throw new Exception(
-          "Set of documents found not equal for words " + Arrays.asList(words).toString() + " "
-              + documentsFoundInIndex + " " + documentsFoundByGrep);
+      throw new Exception("Set of documents found not equal for words " + Arrays.asList(words) + " "
+          + documentsFoundInIndex + " " + documentsFoundByGrep);
     }
 
-    log.debug("Grep and index agree " + Arrays.asList(words).toString() + " "
-        + documentsFoundInIndex.size());
+    log.debug("Grep and index agree " + Arrays.asList(words) + " " + documentsFoundInIndex.size());
```

Comment:
```suggestion
    log.debug("Grep and index agree " + Arrays.toString(words) + " " + documentsFoundInIndex.size());
```
Does the same thing without having to first convert it to List.